hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -283,6 +283,7 @@ final void initInvocationTarget() throws Exception {
             if (previousTargetMember != null) {
                 // If a target member was found earlier but current target member is null
                 // then it means a member left.
+                // TODO: not always true ??
```

Comment:
TODO?

Reply:
hmm this is a left-over comment from a CP testing, not relevant for this PR. but still the comment is true...